style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+#header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.my-logo {
+  text-decoration: none;
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+}
+
+body {
+  background: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+}
```

Comment:
- [ ] It appears that the background you have chosen and the Figma background are not consistent. According to the project requirements, it is necessary to download and utilize the Figma background for your project. Currently, I observe that you are solely using a background color instead of incorporating the downloaded Figma background image. Please ensure that you integrate the appropriate Figma background image as specified.